From f2aea4409927995e9e8961418bacd544402f05f1 Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Mon, 6 May 2019 21:22:21 -0600
Subject: [PATCH] [SQUASH][DNM] Merge tag 'android-9.0.0_r37' into
 staging/lineage-16.0_merge-android-9.0.0_r37

Android 9.0.0 Release 37 (PQ3A.190505.002)

* tag 'android-9.0.0_r37':
  Import translations. DO NOT MERGE

https://github.com/haggertk/android_packages_apps_ExactCalculator/tree/lineage-16.0_merge-android-9.0.0_r37

Change-Id: I6c17834d78b8505e81f4cd6395ad93d2df582ef4
---
 res/values-as/strings.xml | 4 ++--
 res/values-es/strings.xml | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/res/values-as/strings.xml b/res/values-as/strings.xml
index 017c071..6592901 100644
--- a/res/values-as/strings.xml
+++ b/res/values-as/strings.xml
@@ -22,7 +22,7 @@
     <string name="mode_deg" msgid="1146123354434332479">"ডি."</string>
     <string name="mode_rad" msgid="1434228830085760996">"ৰে."</string>
     <string name="clr" msgid="6730945431543327337">"clr"</string>
-    <string name="cleared" msgid="3952521190281880880">"মচা হ\'ল"</string>
+    <string name="cleared" msgid="3952521190281880880">"মচা হ’ল"</string>
     <string name="del" msgid="5878069000864178910">"del"</string>
     <string name="desc_const_e" msgid="1889187337970539507">"অইলাৰৰ নম্বৰ"</string>
     <string name="desc_const_pi" msgid="5430918714009441655">"পাই"</string>
@@ -67,7 +67,7 @@
     <string name="error_overflow" msgid="7800547394563434764">"অসীম?"</string>
     <string name="error_zero_divide" msgid="458040988686661801">"০ৰে হৰণ কৰিব নোৱাৰি"</string>
     <string name="text_copied_toast" msgid="2104264466812485425">"পাঠৰ প্ৰতিলিপি কৰা হৈছে"</string>
-    <string name="cancelled" msgid="2207323593183640994">"গণনা বাতিল কৰা হ\'ল।"</string>
+    <string name="cancelled" msgid="2207323593183640994">"গণনা বাতিল কৰা হ’ল।"</string>
     <string name="timeout" msgid="5461892570729418573">"মান অসীম বা অনিৰ্দিষ্ট হ\'ব পাৰে।"</string>
     <string name="ok_remove_timeout" msgid="8344529153919268011">"গণনাৰ বাবে নিৰ্ধাৰিত কৰা সময় বঢ়াওক"</string>
     <string name="dismiss" msgid="7872443888515066216">"বাতিল কৰক"</string>
diff --git a/res/values-es/strings.xml b/res/values-es/strings.xml
index 7a9da71..cb7dcb9 100644
--- a/res/values-es/strings.xml
+++ b/res/values-es/strings.xml
@@ -70,7 +70,7 @@
     <string name="cancelled" msgid="2207323593183640994">"Cálculo cancelado."</string>
     <string name="timeout" msgid="5461892570729418573">"Es posible que el valor sea infinito o no esté definido."</string>
     <string name="ok_remove_timeout" msgid="8344529153919268011">"Utilizar tiempos de espera más largos"</string>
-    <string name="dismiss" msgid="7872443888515066216">"Descartar"</string>
+    <string name="dismiss" msgid="7872443888515066216">"Cerrar"</string>
     <string name="exact" msgid="2597237880041789948">"(exacto)"</string>
     <string name="approximate" msgid="7117143366610670836">"(±1 en el último dígito)"</string>
     <string name="menu_leading" msgid="2338520833272667740">"Responder con dígitos iniciales"</string>
-- 
2.17.1

